cmd/influxd/main.go

```diff
@@ -45,6 +45,9 @@ func main() {
 		commit = "unknown"
 	}
 
+	// Set parallelism.
+	runtime.GOMAXPROCS(runtime.NumCPU())
```

Comment:
I'd like to suggest we explicitly log this, so the user knows it's happening. E.g.
```
log.Println("GOMAXPROCS set to", runtime.GOMAXPROCS(0))
```